### PR TITLE
Added aggregated assignments view

### PR DESCRIPTION
**Pull request by recluze**
To view all students' assignment in one page (to find any obvious mistakes).

### CODE REVIEW

File: AutoGrade/templates/admin/assignment_aggregate_report.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+</div>
+
+<script>
+$(document).ready(function(){
+    $('pre code').each(function(i, block) {
+            hljs.highlightBlock(block);
+    });
+});
+</script>
+<div id="codeModal" class="modal fade" role="dialog">
```
Comment (soachishti) [inline AutoGrade/templates/admin/assignment_aggregate_report.html:76]: It look like this modal is not used.

**Review by soachishti — APPROVED** [review]
Looks good. I don't find any conflict.

File: AutoGrade/urls.py [hunk 2]
```diff
@@ -4,16 +4,17 @@
...
     url(r'^logout/$', auth_views.logout, {'next_page': 'login'}, name='logout'),
     url(r'^signup/$', views.signup, name='signup'),
     url(r'^api/(?P<action>[0-9a-zA-Z_]+)$', views.api, name='api'),
-    
+
     url(r'^assignment_report/(?P<assignment_id>[0-9a-zA-Z_]+)$', views.assignment_report, name='assignment_report'),
+    url(r'^assignment_aggregate_report/(?P<assignment_id>[0-9a-zA-Z_]+)$', views.assignment_aggregate_report, name='assignment_aggregate_report'),
     url(r'^moss_submit/(?P<assignment_id>[0-9a-zA-Z_]+)$', views.moss_submit, name='moss_submit'),
     url(r'^moss_view/(?P<assignment_id>[0-9a-zA-Z_]+)$', views.moss_view, name='moss_view'),
-
```
Comment (soachishti) [inline AutoGrade/urls.py:16]: @recluze Which editor do you use? which find out these spaces/tabs? or you run autopep8?

Comment (recluze) [reply AutoGrade/urls.py:16]: It's `atom`. I think I might have a PEP plugin installed for this (not sure).

**Pull request merged**